src/tasks/update-org-settings/update-org-settings-v2/task.json

```diff
@@ -0,0 +1,73 @@
+{
+  "id": "35861345-efcc-48b2-9512-09ae16529a4d",
+  "name": "PowerPlatformUpdateOrgSettings",
+  "friendlyName": "Power Platform Update Org Settings",
+  "description": "Power Platform Update Org Settings",
+  "author": "Microsoft",
+  "helpMarkDown": "https://aka.ms/buildtoolsdoc Ideas, feedback: https://github.com/microsoft/powerplatform-build-tools/discussions",
+  "category": "Utility",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "demands": [
+    "PowerPlatformTool"
+  ],
+  "version": {
+    "Major": "0",
+    "Minor": "1",
+    "Patch": "0"
+  },
+  "preview": false,
+  "minimumAgentVersion": "1.95.0",
+  "instanceNameFormat": "Power Platform Update Org Settings $(message)",
+  "inputs": [
+    {
+      "name": "authenticationType",
+      "type": "radio",
+      "label": "Authentication type",
+      "defaultValue": "PowerPlatformEnvironment",
+      "required": true,
+      "helpMarkDown": "Authentication via username/password is simpler to set up but won't work if your AAD tenant requires MFA; use Service Principal Name (SPN) if MFA is required.",
+      "options": {
+        "PowerPlatformEnvironment": "Username/password (no MFA support)",
+        "PowerPlatformSPN": "Service Principal/client secret (supports MFA)"
+      }
+    },
+    {
+      "name": "PowerPlatformEnvironment",
+      "type": "connectedService:Generic",
+      "label": "Service connection",
+      "required": true,
+      "visibleRule": "authenticationType = PowerPlatformEnvironment",
+      "helpMarkDown": "Authenticate with your Power Platform environment with username/password. Does not support MFA."
+    },
+    {
+      "name": "PowerPlatformSPN",
+      "type": "connectedService:powerplatform-spn",
+      "label": "Service connection",
+      "required": true,
+      "visibleRule": "authenticationType = PowerPlatformSPN",
+      "helpMarkDown": "Authenticate with your Power Platform environment with an Azure AppID, tenantID and client secret."
+    },
+    {
+      "name": "Name",
+      "label": "Setting Name",
+      "type": "string",
+      "required": true,
+      "helpMarkDown": "Name of the setting to update."
+    },
+    {
+      "name": "Value",
+      "label": "Setting Value",
+      "type": "string",
+      "required": false,
+      "helpMarkDown": "Value to which setting should be updated to."
```

Comment:
Can we add sentence: "Use true/false for Boolean values"? Customers are trying to pass yes/no